h/static/styles/app.scss

```diff
@@ -102,3 +102,38 @@ body {
     top: 1em;
   }
 }
+
+.loggedout-message {
```

Comment:
I suggest we extract this block into its own .scss file and then `@include` it at the components section at the top.